Jenkinsfile

```diff
@@ -128,6 +122,21 @@ ansiColor('xterm') {
           }
 
           if (currentBuild.result == 'SUCCESS'){
+            stage('Bump version'){
+              sh '''#!/bin/bash -ex
+              source ~/.nvm/nvm.sh
+              nvm use v6
+              npm version patch
+              grep "version" package.json | head -1 | awk -F: '{ print $2 }' | sed 's/[",]//g' > .version
```

Comment:
There are some good version bump npm packages that we could use if we don't want to use this magic, lol